pskb_website/static/js/utils.js

```diff
@@ -150,16 +150,6 @@ function create_responsive_tables(div) {
     tables.addClass('table');
 }
 
-/* Change all external links to open in a new tab/window */
-function create_external_links(id) {
```

Comment:
Just for reference, this code is handled in the pskb_website/static/js/marked_settings.js file now.